src/Core/Domain/Product/Image/Command/ProductImageSetting.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+declare(strict_types=1);
+
+namespace PrestaShop\PrestaShop\Core\Domain\Product\Image\Command;
+
+use PrestaShop\PrestaShop\Core\Domain\Product\Image\ValueObject\ImageId;
+use PrestaShop\PrestaShop\Core\Domain\Shop\ValueObject\ShopId;
+
+class ProductImageSetting
+{
+    /**
+     * @var ImageId
+     */
+    private $productImageId;
+    /**
+     * @var array<ShopId>
```

Comment:
should be either
1. when its simple array indexed from zero
```suggestion
     * @var ShopId[]
```
or 
2. (first item is the key, second is the value)
```suggestion
     * @var array<int, ShopId>
```